consensus/istanbul/core/commit.go

```diff
@@ -113,11 +113,17 @@ func (c *core) handleCheckedCommitForPreviousSequence(msg *istanbul.Message, com
 		logger.Debug("Received a commit message for the previous sequence with an unexpected hash", "expected", headBlock.Hash().String(), "received", commit.Subject.Digest.String())
 		return errInconsistentSubject
 	}
-	return c.acceptParentCommit(msg, commit.Subject.View)
+
+	// Add the ParentCommit to current round state
+	if err := c.current.AddParentCommit(msg); err != nil {
```

Comment:
In `acceptParentCommit` we used to log the view of the parent commit, I assume we  no longer care about that?

Reply:
Done

Now using:
```go
logger := c.newLogger("func", "handleCheckedCommitForPreviousSequence", "tag", "handleMsg", "msg_view", commit.Subject.View)
```